server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -197,7 +193,6 @@ private String uploadFile(String objectKey, String filePath) {
     public void copyUtilScripts() throws IOException {
         String resourcePath = "/cloudformation/scripts/util/";
         String outputPath = "output/scripts/util/";
```

Comment:
You could use the static string from the AbstractLifecycle `public static final String UTIL_DIR_PATH`